auditbeat/tests/system/test_base.py

```diff
@@ -47,3 +47,25 @@ def test_template(self):
         assert exit_code == 0
         assert self.log_contains('Loaded index template')
         assert len(es.cat.templates(name='auditbeat-*', h='name')) > 0
+
+    @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
+    def test_dashboards(self):
+        """
+        Test that the dashboards can be loaded with `setup --dashboards`
+        """
+
+        kibana_dir = os.path.join(self.beat_path, "_meta", "kibana")
+        shutil.copytree(kibana_dir, os.path.join(self.working_dir, "kibana"))
+
+        es = Elasticsearch([self.get_elasticsearch_url()])
+        self.render_config_template(
+            modules=[{
+                "name": "auditd",
```

Comment:
I don't think these test get executed because the config for the one above is out-of-date.

Let's use the `file_integrity` module instead of `auditd` because it can run on more platforms. 

```
             modules=[{
                 "name": "file_integrity",
                 "extras": {
                     "paths": ["file.example"],
                 },
```

Reply:
Changed to your suggested config. I assume the first paragraph was related to the second part with the new config to enable it?

Reply:
@exekias In general I wonder if we should require a module to run the `setup` command?